### DISCUSSION TITLE
EPG (Program Guide) update period is 24 hours but from when?

### DISCUSSION THREAD

**Discussion by aasimenator** [rabilrbl/jiotv_go#375] [General]
Hi,

The documentation says that EPG updates every 24 hours and provides a guide for a 2-day duration, but doesn't say when in a 24 hour?
does it run every 24 hours since the first time it was generated, e.g., assume 13th August 2024 at 12 PM
or
does it run every 24 hours, at 12 AM Midnight, regardless of the first time it was generated?

The reason i am asking is this:
I have just managed to add this as a Docker app on Unraid, I would like to create a script that copies the epg and m3u playlist to my JellyFin so that I can mount these in the JellyFin app automatically rather than doing it manually everyday.

Thanks

**Comment by rabilrbl** [comment]
EPG update is done at random time in midnight, you'll see the time in logs.
use `JIOTV_DEBUG=true` if u want the log to be displayed in terminal
